client/README.md

```diff
@@ -0,0 +1,36 @@
+# Name Request Online Frontend
+
+####This project is still very early in its development.  WIP.
+
+## Project setup
+```
+npm install
+```
+
+### Compiles and hot-reloads for development
+```
+npm run serve
+```
+
+### Compiles and minifies for production
+```
+npm run build
+```
+
+### Run your tests
+```
+npm run test
+```
+
+### Lints and fixes files
+```
+npm run lint
```

Comment:
You might want to add a script entry to run lint without changing your files automatically:
`"lint:nofix": "vue-cli-service lint --no-fix",`